src/lib/dl/range.c

```diff
@@ -140,7 +140,7 @@ static bool range_add(zckRange *info, zckChunk *chk, zckCtx *zck) {
     return true;
 }
 
-void PUBLIC zck_range_free(zckRange **info) {
+void ZCK_PUBLIC_API zck_range_free(zckRange **info) {
```

Comment:
In the other files, you have ZCK_PUBLIC_API before the type of the function.  Is there a reason this file is different?  I would prefer to use this ordering (`type ZCK_PUBLIC_API func()`) if it will compile under Windows.

Reply:
I just replace `PUBLIC` with `ZCK_PUBLIC_API` in most places. I can take a look to change all to your preferred ordering.